load-testing/config/load_test_manifest_reader.go

```diff
@@ -72,5 +78,23 @@ func ParseLoadTestManifest(manifestContent []byte) (*LoadTestManifestYAML, error
 		}
 	}
 
+	if parsedManifest.PersistentChain {
```

Comment:
For ephemeral chains (non persistent chains), we need to explain why we don't configure suppliers.

#PUC